avocado/core/utils.py

```diff
@@ -34,3 +34,11 @@ def prepend_base_path(value):
         dist = get_distribution('avocado-framework')
         return os.path.join(dist.location, 'avocado', expanded)
     return expanded
+
+
+def system_wide_or_base_path(value):
+    """Returns either a system wide absolute path, or a relative to the base."""
+    system_wide = os.path.join(os.path.sep, value)
```

Comment:
From this I gather that `value` is a string and potentially path. What if the path is already absolute? I know `//some/path` is still going to work out but a more elegant way would be to just check if the path is absolute.

Reply:
> I could run it tomorrow, I hope it is not too late for you. I left a few comments from your proposed solution for now.

I'll send a v2 and wait for your testing Tomorrow.  Thanks!

Reply:
Even a push here should be fine I think. It is not that many changes after all.

Reply:
> Even a push here should be fine I think. It is not that many changes after all.

Sure... we really need to standardize the development workflow though.  I'll look into some of the tooling we talked about next sprint.

Reply:
No worries about that, this is just a temporary suggestion for the current PR.